chapters/en/chapter10/1.mdx

```diff
@@ -0,0 +1,19 @@
+# Introduction to Argilla[[introduction-to-argilla]]
+
+In Chapter 5 you learnt how to build a dataset using the 🤗 Datasets library and in Chapter 6 you explored how to fine-tune models for some common NLP tasks. In this chapter, you will learn how to use Argilla to **annotate and curate datasets** that you can use to train and evaluate your models.
+
+The key to training models that perform well is to have high-quality data. Although there are some good datasets in the Hub that you could use to train and evaluate your models, these may not be relevant for your specific application or use case. In this scenario, you may want to build and curate a dataset of your own. Argilla will help you to do this efficiently.
```

Comment:
Might be good to link datasets from the Hub here, it helps for people to see examples of the datasets that they'll annotate etc

Reply:
This is still open I think - we should link some datasets here.

Reply:
What kind of datasets do you think might be useful in this introduction?